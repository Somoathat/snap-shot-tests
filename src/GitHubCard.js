import React from 'react'
import Button from 'react-bootstrap/Button';
import Card from 'react-bootstrap/Card';

// username
// photo
// blurb
function GitHubCard() {
  return (
    <Card style={{ width: '18rem' }}>
    <Card.Img variant="top" src="/photo.jpg" />
    <Card.Body>
      <Card.Title>Somoathat</Card.Title>
      <Card.Text>
        A student at SDSU, becoming a programmer.
        
      </Card.Text>
      {/* <Button variant="primary">Go somewhere</Button> */}
    </Card.Body>
  </Card>
  )
}

export default GitHubCard