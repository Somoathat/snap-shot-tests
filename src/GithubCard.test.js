import renderer from 'react-test-renderer'
import { render, screen } from '@testing-library/react';
import GitHubCard from './GitHubCard';

test('renders a snapshot', () => {
  const treeOriginal = renderer.create(<GitHubCard />)
  const tree = treeOriginal.toJSON()
  expect(tree).toMatchSnapshot()
})

