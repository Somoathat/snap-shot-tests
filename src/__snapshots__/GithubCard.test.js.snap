// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders a snapshot 1`] = `
<div
  className="card"
  style={
    Object {
      "width": "18rem",
    }
  }
>
  <img
    className="card-img-top"
    src="/photo.jpg"
  />
  <div
    className="card-body"
  >
    <div
      className="card-title h5"
    >
      Somoathat
    </div>
    <p
      className="card-text"
    >
      A student at SDSU, becoming a programmer.
    </p>
  </div>
</div>
`;
